undertow/src/main/java/org/wildfly/extension/undertow/Host.java

```diff
@@ -94,6 +94,7 @@
     private final int defaultResponseCode;
     private volatile HttpHandler rootHandler = null;
     private volatile AccessLogService  accessLogService;
+    private volatile Function<HttpHandler, HttpHandler> activeRequestTrackerHandler;
```

Comment:
UnaryOperator<HttpHandler> ?

Reply:
Can you clarify?

Reply:
Generally, we would use java.util.function.UnaryOperator to represent decorator functions.

Reply:
Updated.